examples/module1/interface/main.go

```diff
@@ -6,6 +6,20 @@ type IF interface {
 	getName() string
 }
 
+type Fruit interface {
+	Describe() string
+}
+
+type Apple struct {
+	Color string
+	Shape string
+	Taste string
+}
+
+func (a Apple) Describe() string {
+	return fmt.Sprintf("Apple: the color is %s, the shape is %s, the taste is %s", a.Color, a.Shape, a.Taste)
```

Comment:
For better performance!
```suggestion
	var sb strings.Builder
        sb.WriteString("Apple: the color is ")
        sb.WriteString(a.Color)
        sb.WriteString(", the shape is ")
        sb.WriteString(a.Shape)
        sb.WriteString(", the taste is ")
        sb.WriteString(a.Taste)
     return sb.String()
```